### PR TITLE
Power saw recipes

### PR DESCRIPTION
**Pull request by krulunio**
#### Summary
Content "Adds/changes recipes to use various power saws"

#### Purpose of change
I started adding more basecamp types and wanted to use furniture tools, so I added fake power tools inside furniture, but some of those don't fit into existing recipes, so I decided to add/change some recipes.

#### Describe the solution
Change existing recipes if they should use power saws, or add new recipes.

### CODE REVIEW

**Review by Maleclypse — COMMENTED** [review]
I want to double check that eliminating the "from logs" and "from wooden beams" recipes is part of your intention rather than you adding in the new "with_planer" and "with_power_saw" recipes in addition to the existing two?

**Comment by krulunio** [comment]
Yes, removing those recipes is part of my intention, because comments imply that those recipes should eventually change.

File: data/json/recipes/other/parts_construction.json [hunk 1]
```diff
@@ -18,17 +18,15 @@
     "type": "recipe",
     "activity_level": "fake",
     "result": "wood_beam",
-    "id_suffix": "from logs",
```
Comment (I-am-Erk) [inline data/json/recipes/other/parts_construction.json:21]: These should be new recipes, not replacing the old ones. People still need the option to use hand tools.

Comment (krulunio) [reply data/json/recipes/other/parts_construction.json]: Ok, I reverted changes and added my recipes as alternatives

**Review by I-am-Erk — APPROVED** [review]
I think this is ready once the linting and formating errors are resolved.

**Pull request merged**